botcity/core/bot.py

```diff
@@ -13,6 +13,10 @@
 from botcity.base.utils import is_retina, only_if_element
 from PIL import Image
 
+from pywinauto.application import Application, WindowSpecification
```

Comment:
This must be conditionally imported to avoid issues with `non-windows` hosts.

Reply:
Now I'll take a look, because the test complained about the return type of the method that is in the core...

Reply:
Maybe put it in quotes `'Application'`.

Reply:
I think the changes on `setup.py` look good. The usage of remove was correct. I messed up in my head thinking that `pop` was not index based.